modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -161,6 +199,124 @@ public int call(final String[] args, final Cfg cfg) {
                         }
                     }
                     break;
+
+                case "-n":
+                case "--network":
+                    if ( (args.length == 2 || args.length == 4) && isValid(args[1])) {
+
+                        switch (args[1].toLowerCase()) {
+                            case "mainnet":
```

Comment:
Preefer to switch to enums instead of strings, they're more self-defining.

Reply:
will add enum onto implementation